ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/model/PostConditionsNode.java

```diff
@@ -205,7 +205,7 @@ public PostTopicNode(
         @Override
         public void describeTo(final Description description) {
           try {
-            description.appendText(TestJsonMapper.INSTANCE.get().writeValueAsString(this));
+            description.appendText(TestJsonMapper.INSTANCE.get().writeValueAsString(PostTopicNode.this));
```

Comment:
Is this just for clarity? `this` should already refer to `PostTopicNode.this` here, right?

Reply:
no, this is within the `Matcher` anonymous class. I need to use `PostTopicNode.this` to refer to the outer wrapper (non-anon) class